api_types.ts

```diff
@@ -338,6 +338,40 @@ export interface ParamDef<T extends UnionType> {
    */
   suggestedValue?: SuggestedValueType<T>;
 
+  /**
+   * In certain contexts, Coda's default behavior is to crawl *all* data available, in which case
+   * the {@link ParamDef.suggestedValue} will not be ideal, as most use cases will prefer efficiency
+   * over completeness. Use this field to specify a default value when Coda is crawling all data.
+   *
+   * @hidden
+   */
+  // TODO(patrick): Unhide this
+  fullCrawlSuggestedValue?: SuggestedValueType<T>;
```

Comment:
Reasonable! It would be a little messy with types, needing to pass down the `<T>` but if @jonathan-codaio agrees with you I'm happy to move it.

Reply:
Oh interesting. I guess I had read this as meaning "suggested value when used in a Brain ingestion" but with a less Brain-y name. Is the usage of this truly only specific to when there's a crawl involved? If so, yeah it would seem to make sense to group it with the crawl stuff.

Reply:
No, your reading is correct, but I think the question are "ingestions" and "crawls" the same thing or not?

Reply:
I'm fine either way on that question, I just think if we stick with `crawl` in the name of this new field it should probably live under `crawlStrategy`.

Reply:
I don't see ingestions and crawls as the same. Crawls are a particular implementation strategy that a pack may choose to use.

Agree that having `crawl` in both places with different meanings is probably undesirable.